artemis-server/src/test/java/org/apache/activemq/artemis/tests/util/ActiveMQTestBase.java

```diff
@@ -171,6 +171,9 @@
    @ClassRule
    public static ThreadLeakCheckRule leakCheckRule = new ThreadLeakCheckRule();
 
+   @Rule
+   public NoFDBehind noFDBehind = new NoFDBehind(-1, 1000);
```

Comment:
Whats FD, i can guess due to context of this PR, but the point is, i shouldnt have to guess and in future without PR context it will be harder to guess, please avoid using acronym or shorthand, for others readability.

Reply:
I renamed noFDBehind to noProcessFilesBehind.

Reply:
@michaelandrepearce I would have thought that FD is a common usage for File Descriptors?